### ISSUE TITLE
Why is this git so unresponsive?

### ISSUE THREAD

**Issue by subratastack** [dschnelldavis/angular2-json-schema-form#263]
I'm posting many issues but the contributors hardly reply. Disappointed with such a nice git.

**Comment by jscharett** [comment]
@dschnelldavis Are you still maintaining this repo?  There is a growing list of users who are actively adding issues/PR's but not seeing any movement on them.  If your no longer maintaining this, can we see about transfering ownserhip?  Maybe see if the @json-schema-form group is interested in taking it?

**Comment by RichardMViGEM** [comment]
My guess is that @dschnelldavis is busy with the newer, ng6-compatible version of this repo, over at
https://github.com/catull/ngx-json-schema

**Comment by jscharett** [comment]
Thast's just someone else's fork, not a newer version.  I believe @shamoons is going to try and move forward with his fork here https://github.com/shamoons/ng-json-schema-form

**Comment by shamoons** [comment]
@jscharett that's correct. I'm going to move forward with it at https://github.com/shamoons/ng-json-schema-form

I'm in the process of reviewing existing PR's currently.

**Comment by catull** [comment]
@RichardMViGEM Nice try.

That repo is a fork, @dschnelldavis has no involvement with [`ngx-json-schema`](https://github.com/catull/ngx-json-schema).
@dschnelldavis has build a good library, I am not taking any credit for it.

[`ngx-json-schema`](https://github.com/catull/ngx-json-schema) is

- `angular2-json-schema-form` rebooted
- "free" of `Angular Material Design` (don't ask for supporting it)
- packaged with [`ng-packagr`](https://www.npmjs.com/package/ng-packagr); thus it can be used in Angular, React, ES5, ES6, ES2015 ....
- usable from Angular 6+, not really usable for Angular < 6
- available from npmjs.org: [`ngx-json-schema`](https://www.npmjs.com/package/https://www.npmjs.com/package/ngx-json-schema)

I am offering to contribute to @shamoons's repo, glad he has more time to dedicate to this library.

**Comment by neil-coutinho** [comment]
@shamoons After installing your package via npm (npm install ng-json-schema-form) there are alot of error when you load the module.
Running 
CLI:  6.0.3
Node: 8.9.4
Angular: 6.0.2
Typescript: 2.7.2
Webpack 4.8.3

![screen shot 2018-07-12 at 2 59 30 pm](https://user-images.githubusercontent.com/33371261/42625191-beabf8ca-85e4-11e8-8634-fd2657bbb330.png)

**Comment by jscharett** [comment]
Not sure if this is the issue, but your using Angular 6.  I don't think @shamoons fork has been updated to support Angular 6 yet.

**Comment by jscharett** [comment]
@shamoons Looking at the deployed package that's coming down for ng-json-schema-form, it looks like it may not be compiled correctly?  Its def different than the deployed package for this repo.  Any reason for that?

**Comment by shamoons** [comment]
I'll take a look this weekend

**Comment by shamoons** [comment]
I enabled issues on https://github.com/shamoons/ng-json-schema-form
